### ISSUE TITLE
Forge 1.18.2 Issues

### ISSUE THREAD

**Issue by Bananalord57** [QuantumXenon/origins-plus-plus#83]
**Loader**
Forge

**Versions**
1.18.2 forge (40.1.76)
Origins forge - 1.18.2- 1.5.0.1/1.5.0.0
Origins++ - 1.5.3
Pehkui forge - 1.18.2-3.4.2
Caelus forge 1.18,2-3.0.0.2

**While using Origins forge 1.5.0.1**
Crashes when selecting some classes such as the corrupted wither
[crash 1.5.0.1.txt](https://github.com/QuantumXenon/origins-plus-plus/files/9611975/crash.1.5.0.1.txt)
most of the times worlds will corrupt when generating them resulting in a java.util.ConcurrentModificationExeption (this can also happen while using 1.5.0.0 but is less prevalent)

**While using Origins forge 1.5.0.0**
Some origin abilities are bound to the same key bind
None of the corrupted wither's abilities function and the player is earraped with the beacon power up sound
Spike throw on the ice porcupine class doesn't have a cooldown and the arrows spawned can be collected
Mischievous on the mouse class seems to teleport the player a random distance with no cooldown
Artificial construct can't throw tipped arrows
Illusionary clones immediately disappear
etc.
(I suspect there's an underlying issue causing most of these, I can add more if you'd like me to)
edit:
I've noticed that versions of Origins++ after 1.5.3 appear to have the same issues as the corrupted wither so I'll just assume they also contain the other issues mentioned, also 1.7-1.7.3 are tagged with 1.18.2 on curse forge but don't appear to work for 1.18.2

**Comment by QuantumXenon** [comment]
@Bananalord57 Thanks for all the reports!

The keybinds issue was a problem with Origins (Forge) and has been fixed in 1.5.0.6 (for 1.19.2)

The CurseForge versioning isn't quite right because I locked the Forge version to 1.19.2 to avoid the keybind issue (as it is very major), but it still works as intended for Fabric. I could split them again if there is no Origins (Forge) update for 1.18.2 that fixes the keybinds.

I'll have a look at the other issues you have mentioned.
